src/tests/ftest/util/dmg_utils.py

```diff
@@ -334,18 +334,18 @@ def storage_query_target_health(self, rank, tgtid):
         return self._get_result(
             ("storage", "query", "target-health"), rank=rank, tgtid=tgtid)
 
-    def storage_query_nvme_health(self):
-        """Get the result of the 'dmg storage query nvme-health' command.
+    def storage_scan_nvme_health(self):
+        """Get the result of the 'dmg storage scan --nvme-health' command.
 
         Returns:
             CmdResult: an avocado CmdResult object containing the dmg command
                 information, e.g. exit status, stdout, stderr, etc.
 
         Raises:
-            CommandFailure: if the dmg storage prepare command fails.
+            CommandFailure: If dmg storage scan --nvme-health command fails.
 
         """
-        return self._get_result(("storage", "query", "nvme-health"))
+        return self._get_result(("storage", "scan", "--nvme-health"))
```

Comment:
So since `nvme-health` is being moved to the StorageScan class as a formatted parameter, the best way to call it would be by passing it into `get_result` as an argument: 

```
return self._get_result(("storage", "scan"), nvme_health=True)
```